articles/contributing-style-guide.md

```diff
@@ -2,59 +2,58 @@
 author: cgranade
 description: Learn the naming, input, documentation and formatting conventions for Q# programming language applications and libraries.
 ms.author: chgranad
-ms.date: 02/01/2021
+ms.date: 06/27/2022
 ms.service: azure-quantum
 ms.subservice: qdk
 ms.topic: conceptual
 no-loc: ['Q#', '$$v']
 title: Q# Style Guide
 uid: microsoft.quantum.contributing-qdk.overview.style
+ms.custom: kr2b-contr-experiment
 ---
 
 # Q# style guide #
 ## General conventions ##
 
-The conventions suggested in this guide are intended to help make programs and libraries written in Q# programming language easier to read and understand.
+The conventions in this guide make it easier to read and understand programs and libraries written in the Q# programming language.
 
 ## Guidance
 
-We suggest:
+We recommend:
 
-- Never disregard a convention unless you’re doing so intentionally in order to provide more readable and understandable code for your users.
+- Stick to convention unless you intentionally want to provide code that is more readable and understandable for users.
```

Comment:
I'm not sure I entirely agree with this edit; makes it sound like breaking convention is _in general_ a way to provide more readable and understandable code, but that's the exception rather than the norm.